hypothesis-python/tests/cover/test_lambda_formatting.py

```diff
@@ -183,3 +183,18 @@ def test_can_handle_nested_lambda_in_decorator_argument():
     assert (
         get_pretty_function_description(decorator_with_wrapper[0]) == "lambda x: x + 1"
     )
+
+
+is_approx_π = lambda x: x == 3.1415
```

Comment:
I think this is python 3 only syntax, so this file will fail to even load on python 2, right?
```
>>> is_approx_π = lambda x: x == 3.1415
  File "<stdin>", line 1
    is_approx_π = lambda x: x == 3.1415
              ^
SyntaxError: invalid syntax
```

Reply:
Yep.  It can easily be moved to `tests/py3/test_unicode_identifiers.py` though :smile: